### DISCUSSION TITLE
Daymet update to V4.5

### DISCUSSION THREAD

**Discussion by scottyhq** [microsoft/PlanetaryComputer#311] [Data request]
The Daymet dataset https://planetarycomputer.microsoft.com/dataset/daymet-daily-na claims to be 4.0, but according to the DOI (https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1840) it is actually 4.2

The latest release (https://doi.org/10.3334/ORNLDAAC/2129) has data through 2022

**Comment by TomAugspurger** [comment]
> The Daymet dataset https://planetarycomputer.microsoft.com/dataset/daymet-daily-na claims to be 4.0, but according to the DOI (https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1840) it is actually 4.2

Probably an oversight on my part last time I updated it.

> The latest release (https://doi.org/10.3334/ORNLDAAC/2129) has data through 2022

I worked on this a while back but ran into issues with the Zarr conversion or rechunking... I don't recall the details, but probably running out of memory somewhere (I see `hi.zarr` and `pr.zarr`, but no `na.zarr`). I'll pick that up again sometime, but it'll be a little while. That'll give me an excuse to use the new Beam-based pangeo-forge.

One quick question: would you consider a change to the chunking to be "API breaking", or likely to break workflows? I vaguely recall needing to adjust stuff.

**Reply by scottyhq** [reply]
Thanks Tom! From my point of view changing chunking is fine. One other quick note, the data page says that the netCDF files are also available, but I didn't see any examples of how to access them (I thought I'd access one to verify the version).

**Reply by michele-personal** [reply]
Hi.  The most current Daymet daily data is [https://doi.org/10.3334/ORNLDAAC/2129](https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=2129).  2023 is now available.  The ORNL DAAC has changed its versioning policy.  This is now Version 4.1.  Please see [Table 8](https://daac.ornl.gov/DAYMET/guides/Daymet_Daily_V4R1.html) for an explanation.

**Reply by TomAugspurger** [reply]
Thanks for the updated info! I'll make sure we pull the latest data when we get around to updating this.
